docs/docs/cmd/teams/meeting/meeting-transcript-list.md

```diff
@@ -0,0 +1,93 @@
+# teams meeting transcript list
+
+Lists all transcripts for a given meeting.
+
+## Usage
+
+```sh
+m365 teams meeting transcript list [options]
+```
+
+## Options
+
+`-u, --userId [userId]`
+: The id of the user, omit to list meeting transcripts list for current signed in user. Use either `id`, `upn` or `email`, but not multiple.
+
+`--upn [upn]`
+: The upn of the user, omit to list meeting transcripts list for current signed in user. Use either `id`, `upn` or `email`, but not multiple.
+
+`--email [email]`
+: The email of the user, omit to list meeting transcripts list for current signed in user. Use either `id`, `upn` or `email`, but not multiple.
+
+`-m, --meetingId <meetingId>`
+: The id of the meeting.
+
+--8<-- "docs/cmd/_global.md"
+
+## Examples
+
+Lists all transcripts made for the current signed in user and Microsoft Teams meeting with given id
+
+```sh
+m365 teams meeting transcript list --meetingId MSo1N2Y5ZGFjYy03MWJmLTQ3NDMtYjQxMy01M2EdFGkdRWHJlQ
+```
+
+Lists all transcripts made for the _[garthf@contoso.com](mailto:garthf@contoso.com)_ and Microsoft Teams meeting with given id
```

Comment:
I don't think it's very useful to make a mailto link for a fake email.

```suggestion
Lists all transcripts for a meeting of a specific user
```

Reply:
✅